### DISCUSSION TITLE
Welcome email resend

### DISCUSSION THREAD

**Discussion by Raito00** [dani-garcia/vaultwarden#3528] [Ideas]
How to resend Welcome email again ?
Because some time on new user registration have smtp error:
[vaultwarden::api::core::accounts][ERROR] Error sending welcome email: SMTP 4xx error: transient error (454): 4.7.0 Temporary authentication failure: Connection lost to authentication server

And then need delete user and register again to get Welcome email.

**Comment by BlackDex** [comment]
There currently isn't a way to do this. Deleting and re-invite is for now the only way.

**Comment by tessus** [comment]
You can use the `/users/<uuid>/invite/resend` API endpoint and there should be a `Resend invite` link in the admin interface.

see https://github.com/dani-garcia/vaultwarden/pull/3397

**Reply by BlackDex** [reply]
I totally forgot about that new addition.
And there is a re-invite in the organization member interface, but that only works if invited to an org.

**Reply by tessus** [reply]
But if someone is invited not to an org, there's still the default org `Vaultwarden`. Or does this not count? If not, maybe there is way to add it to the user interface of the admin panel as well.

**Reply by BlackDex** [reply]
We can't add that to the user interface of the web-vault, since there are no owners for that org.
But, that is also not needed, since there is that **resend** link in the `/admin` interface. So both scenarios are handled i think.

**Reply by tessus** [reply]
Maybe I haven't expressed myself correctly. I did not mean adding it to the user interface, but the user page in the admin interface. You stated

> there is a re-invite in the organization member interface, but that only works if invited to an org.

But I am not sure why the re-send would be in web-vault. Unlewss bitwarden added it. Anyway, I meant (since I haven't seen the re-send button/link in the admin interface myself, which means I don't know when it shows up) that this link should be presented also when not invited to an org, but invited as a user to your vw instance.

**Reply by BlackDex** [reply]
In the organization interface there is a resend option. Those are useful if you invited someone into an org.

If you didn't, and just invited them via the `/admin` interface, then there is a resend invite button for a non fully created account there.
